src/machinetalk/support/unionread.c

```diff
@@ -39,19 +39,18 @@ bool print_container(pb_istream_t *stream)
     uint64_t taghdr;
     pb_wire_type_t wiretype;
 
-#if 1
     if (!pb_decode_varint(stream, &taghdr)) {
 	printf("Parsing taghdr failed: %s\n", PB_GET_ERROR(stream));
     }
     tag = taghdr >> 3;
     wiretype = taghdr & 0x07;
-#else
+/**********************************************************
```

Comment:
I have no idea if it is any good and I don't think to put it into a comment makes it any better. Can we just remove it from the code? If anyone really needs this piece of code he can find it in the git history.